modules/services/window-managers/fluxbox.nix

```
@@ -0,0 +1,135 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let cfg = config.xsession.windowManager.fluxbox;
+in {
+  meta.maintainers = [ maintainers.AndersonTorres ];
+
+  options = {
+    xsession.windowManager.fluxbox = {
+      enable = mkEnableOption "Fluxbox window manager";
+
+      package = mkOption {
+        type = types.package;
+        default = pkgs.fluxbox;
+        defaultText = literalExpression "pkgs.fluxbox";
+        description = "Package to use for running Fluxbox WM.";
+      };
+
+      init = mkOption {
+        type = types.lines;
+        default = "";
+        description = ''
+          Init configuration for Fluxbox, written to
+          <filename>~/.fluxbox/init<filename>. Look at the fluxbox(1) manpage
+          for details.
+        '';
+      };
+
+      apps = mkOption {
+        type = types.lines;
+        default = "";
+        description = ''
+          Apps configuration for Fluxbox, written to
+          <filename>~/.fluxbox/apps<filename>. Look at the fluxbox(1) manpage
+          for details.
+        '';
+      };
+
+      keys = mkOption {
+        type = types.lines;
+        default = "";
+        description = ''
+          Keyboard shortcuts configuration for Fluxbox, written to
+          <filename>~/.fluxbox/keys<filename>. Look at the fluxbox-keys(1)
+          manpage for details.
+        '';
+      };
+
+      menu = mkOption {
+        type = types.lines;
+        default = "";
+        description = ''
+          Menu configuration for Fluxbox, written to
+          <filename>~/.fluxbox/menu<filename>. Look at the fluxbox-menu(1)
+          manpage for details.
+        '';
+      };
+
+      slitlist = mkOption {
+        type = types.lines;
+        default = "";
+        description = ''
+          Slitlist configuration for Fluxbox, written to
+          <filename>~/.fluxbox/slitlist<filename>. Look at the fluxbox(1)
+          manpage for details.
+        '';
+      };
+
+      windowmenu = mkOption {
+        type = types.lines;
+        default = "";
+        description = ''
+          Window menu configuration for Fluxbox, written to
+          <filename>~/.fluxbox/windowmenu<filename>. Look at the fluxbox-menu(1)
+          manpage for details.
+        '';
+      };
+
+      extraCommandLineArgs = mkOption {
+        type = with types; listOf str;
+        default = [ ];
+        example = [ "-log" "/tmp/fluxbox.log" ];
+        description = ''
+          Extra command line arguments to pass to <command>fluxbox</command>.
+          Look at the fluxbox(1) manpage for details.
+        '';
+      };
+
+      extraStartupCommands = mkOption {
+        type = with types; listOf str;
+        default = "";
+        example = "";
+        description = ''
+          Extra startup commands executed before <command>fluxbox</command>
+          starts.
+
+          Caution! Commands that should keep running after the window manager
```

Unfortunately, to create a paragraph break you need `</para><para>` like so:

```suggestion
          starts.
          </para><para>
          Caution! Commands that should keep running after the window manager
```